spyder/plugins/projects/plugin.py

```diff
@@ -321,8 +318,14 @@ def _create_project(self, root_path, project_type=EmptyProject.ID,
         """Create a new project."""
         project_types = self.get_project_types()
         if project_type in project_types:
-            project = project_types[project_type](root_path, self)
-            project.create_project()
+            project_type_class = project_types[project_type]
+            project = project_type_class(
+                root_path, project_type_class._PARENT_PLUGIN)
+
+            created_succesfully, message = project.create_project()
+            if not created_succesfully:
+                QMessageBox.warning(self, "Project creation", message)
+                # FIXME: What do do in failed creation?

```

Comment:
I think for now you should put a `return` here, so the project creation is aborted at this point.

Reply:
I think that if the plugin reports an un-successful project creation (by pressing for example the cancel button), spyder can go ahead and remove the whole project directory, regardless what is inside.

Reply:
I agree. Let's go with option 1.

Reply:
Fixed